app/ui/cmd/go.mod

```diff
@@ -0,0 +1,13 @@
+module github.com/syndesisio/syndesis
```

Comment:
Could we agree to stay with go 1.10 and eg dep as long the go 1.11 module support is marked as experimental ? Happy to switch if the module support has been proven to be stable in the wild, though, as it will be the future of dependency management. Also it will free us from staying in the GOPATH (this issue is solved in "syndesis" for go 1.10 a bit differently)

Reply:
I’d rather stick with 1.11. Modules are just easier to develop with.

Reply:
Ok, but then we should upgrade the operator also to 1.11 and use the same builder image.

Reply:
However we approach this, please use one baseline for docker based projects that need to productised. This means, same docker version, dependency management, base images, etc